db/migrate/20160303183011_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.integer :role_id
```

Comment:
Instead of adding an `id` column, you can do `t.references :role, index: true` to create the ID column and automatically index the relationship. See http://stackoverflow.com/questions/32021638/rails-4-migration-add-table-with-reference